adb_graphics/default_specs.yml

```diff
@@ -982,7 +982,7 @@ rh: # Relative Humidity
   mean: # Mean RH 850mb to 500mb
     <<: *rh
     print_units: false
-    title: Mean 850-500mb Relative Humidity (%, shaded), 700mb Wind (kt)
+    title: Mean 850-500mb RH (%, shaded), 700mb Wind (kt)
```

Comment:
What does this one do about the units? I was thinking we didn't typically specify units in the title and they were handled by the script.